File: tests/test-fill-stdout.cc
/*
 * Floods stdout and should fail with TIME-LIMIT or RUN-ERROR
 * depending on whether timelimit or filesize limit is reached first.
 *
 * @EXPECTED_RESULTS@: TIME-LIMIT,RUN-ERROR
 */

#include <iostream>
#include <string>

int main()
{
	while ( 1 ) std::cout << "Fill stdout with nonsense, to test filesystem stability.\n";

	return 0;
}

File: example_problems/hello/submissions/time_limit_exceeded/test-fill-stdout.pas
(*
 * Floods stdout and should fail with TIMELIMIT, since output is
 * automatically truncated at the filesize limit.
 *
 * @EXPECTED_RESULTS@: TIMELIMIT
 *)

program fillstdout(input, output);

begin
   While 1=1 Do
   begin
	  writeln('Fill stdout with nonsense, to test filesystem stability.');
   end;
end.
